### PR TITLE
Fix after review

### PR DESCRIPTION
**Pull request by eugene-chr**

### CODE REVIEW

File: pages/movies/[type].js [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
 import { getLocale } from 'utils/getLocale';
 import { fetchMovies } from 'concepts/fetchMovies';
 import { fetchGenres } from 'concepts/fetchGenres';
 
 export const getServerSideProps = async ({ req, query }) => {
   const locale = getLocale(req);
-  // TODO if manual enter to URL query page=0?
-  const movieList = await fetchMovies(query.type, query.page, locale);
+  let movieList;
+  movieList = await fetchMovies(query.type, query.page, locale);
```
Comment (plzen) [inline pages/movies/[type].js:8]: You can write this in one line

`let movieList = await fetchMovies(query.type, query.page, locale);`

File: pages/tv-shows.js [hunk 2]
```diff
@@ -2,11 +2,21 @@ import { fetchTVShows } from 'concepts/fetchTVShows';
...
 export const getServerSideProps = async ({ req, query }) => {
+  /* TODO if we have page query that is larger than totalPages?
+      how we can change url? */
+  let tvShowsResponse;
+  const queryPage = checkQueryPage(+query.page);
   const locale = getLocale(req);
-  const tvShowsResponse = await fetchTVShows(locale, query.page);
-  const tvShows = await tvShowsResponse.json();
-  const { results, total_pages, page } = tvShows;
+  tvShowsResponse = await fetchTVShows(locale, queryPage);
```
Comment (plzen) [inline pages/tv-shows.js:13]: You can define variable here

File: components/People/component.js [hunk 3]
```diff
@@ -4,31 +4,35 @@ import PropTypes from 'prop-types';
...
-          image={profile_path}
-          title={name}
-        />
-      ))}
-    </MediaGrid>
-    <Pagination page={page} totalPages={totalPages} router={router} />
-  </>
-);
+const PeopleComponent = ({ people, totalPages, page }) => {
+  return (
```
[the remark this answers is not in the document]
Comment (eugene-chr) [reply components/People/component.js:11]: done

File: components/shared/ConditionalWrapper/component.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  }
+
+  return isEmpty(identifier);
+};
+
+const ConditionalWrapper = ({ identifier, children, tagName }) => {
+  const condition = check(identifier);
+
+  return (
+    <>
```
[the remark this answers is not in the document]
Comment (eugene-chr) [reply components/shared/ConditionalWrapper/component.js:20]: done

File: components/shared/Pagination/component.js [hunk 5]
```diff
@@ -1,94 +1,33 @@
...
+    <NavButton disabled={page === 1} type="button" onClick={() => navigatePage(page - 1)}>
+      <FormattedMessage id="pagination.prev" />
+    </NavButton>
+    <div>
+      {pagination.map((item) => (
+        <PageButton
+          key={item.key}
+          active={item.page === page}
+          type="button"
+          onClick={() => navigatePage(item.page)}
```
[the remark this answers is not in the document]
Comment (eugene-chr) [reply components/shared/Pagination/component.js:20]: done

File: pages/movies/[type].js (hunk 1 above)
Comment (eugene-chr) [reply pages/movies/[type].js:8]: done

**Pull request merged**